backend/typescript/types.ts

```diff
@@ -1,5 +1,8 @@
 export type Role = "User" | "Admin";
 
+export type Status = "PENDING" | "ACCEPTED";
```

Comment:
Don't need to add anything here, we're getting rid of this file since this is from mongoose